apps/web/src/app.css

```
@@ -0,0 +1,92 @@
+:root {
+  color: #f6efe5;
+  background:
+    radial-gradient(circle at top, rgba(255, 196, 118, 0.22), transparent 38%),
+    linear-gradient(180deg, #1d2638 0%, #0f141d 100%);
+  font-family: "Trebuchet MS", "Verdana", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Verdana" (font-family-name-quotes)

(font-family-name-quotes)